inline.h

```diff
@@ -3023,6 +3023,15 @@ Perl_utf8n_to_uvchr_msgs(const U8 * const s0,
         const U8 * s = s0;
         const U8 * send = s + curlen;
 
+        /* Find which dfa table to use.  If no restrictions, use [0].  If any,
+         * use at least [1]; and use [2] for the most restrictive */
+        const U8 * table =
```

Comment:
`table` isn't modified and could be const (this declaration is for a variable pointer to const)